src/lib/crypto/kimchi_backend/common/plonk_constraint_system.ml

```diff
@@ -308,6 +308,7 @@ module Plonk_constraint = struct
           }
       | AddFixedLookupTable of { id : int32; data : 'f array array }
       | AddRuntimeTableCfg of { id : int32; first_column : 'f array }
+      | RuntimeLookup of { table_id : 'v; idx : 'v; v : 'v }
```

Comment:
Shouldn't this just be called `Lookup`? It'll be used for fixed tables too.

Reply:
There is already a Lookup constraint for fixed tables.

Reply:
Can you please combine them? Uses don't care, and this breaks the ability to use partially-fixed tables.

Reply:
Fixed in de72520412413eed6469dce74f1d92a64e5578fa